avocado/utils/memory.py

```diff
@@ -115,7 +115,7 @@ def read_from_meminfo(key):
     :param key: Key name, such as ``MemTotal``.
     """
     cmd_result = process.run('grep %s /proc/meminfo' % key, verbose=False)
-    meminfo = cmd_result.stdout
+    meminfo = cmd_result.stdout_text
```

Comment:
It is a function 
   def stdout_text(self):
        if type(self.stdout) in string_types:
            return self.stdout
        if hasattr(self.stdout, 'decode'):
            return self.stdout.decode(self.encoding)
        raise TypeError("Unable to decode stdout into a string-like type")

Reply:
updated